framework/aster-frame/src/vm/space.rs

```diff
@@ -88,6 +89,7 @@ impl VmSpace {
         let mut start_va = range.start;
         let page_size = (range.end - range.start) / PAGE_SIZE;
         let mut inner = self.memory_set.lock();
+        start_va.checked_add(PAGE_SIZE*page_size).ok_or(Error::Overflow)?;
```

Comment:
Rename the `page_size` to `nr_pages` or `num_pages`. The expression of `PAGE_SIZE * page_size` looks illogical.